### ISSUE TITLE
No Response in Home

### ISSUE THREAD

**Issue by RoyHobbs99** [grzegorz914/homebridge-xbox-tv#25]
Hello, I'm attempting to get this plugin to work for the first time, but Home shows no response.

I see this in the logs...
[Homebridge] [5/12/2020, 11:13:52] [XboxTv] Device: 10.0.0.62 XboxOne, state: Online
[Homebridge] [5/12/2020, 11:13:52] [XboxTv] Device: 10.0.0.62 XboxOne, get current App successful: Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application

Any ideas?  I'm mainly just looking to be able to power on/off my Xbox with homekit.

This is my json
    {
            "platform": "XboxTv",
            "plugin_map": {
                "plugin_name": "homebridge-xbox-tv"
            },
            "devices": [
                {
                    "name": "XBOXONE",
                    "host": "10.0.0.62",
                    "xboxliveid": "FD00B581C41F****",
                    "volumeControl": false,
                    "switchInfoMenu": false,
                    "inputs": [
                        {
                            "name": "Minecraft",
                            "reference": "Microsoft.MinecraftUWPConsole_8wekyb3d8bbwe!App"
                        },
                        {
                            "name": "Bluray",
                            "reference": "Microsoft.BlurayPlayer_8wekyb3d8bbwe!Xbox.BlurayPlayer.Application"
                        }
                    ]
                }

**Comment by grzegorz914** [comment]
Are U enabled Instant Power Mode in Xbox settings ?

Any log after pressing power on/off in HomeKit?

**Comment by RoyHobbs99** [comment]
Instant Power Mode is ON
When I long press I don't see any of the input listed in the home app.

All I see in the logs is a lot of this...
[Homebridge] [5/12/2020, 11:51:57] [XboxTv] Device: 10.0.0.62 XBOXONE, get current App successful: Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application
[Homebridge] [5/12/2020, 11:52:27] [XboxTv] Device: 10.0.0.62 XBOXONE, get current App successful: Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application
[Homebridge] [5/12/2020, 11:52:57] [XboxTv] Device: 10.0.0.62 XBOXONE, get current App successful: Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application
[Homebridge] [5/12/2020, 11:53:27] [XboxTv] Device: 10.0.0.62 XBOXONE, get current App successful: Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application
[Homebridge] [5/12/2020, 11:53:57] [XboxTv] Device: 10.0.0.62 XBOXONE, get current App successful: Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application
[Homebridge] [5/12/2020, 11:54:27] [XboxTv] Device: 10.0.0.62 XBOXONE, get current App successful: Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application

**Comment by grzegorz914** [comment]
Check this

```
{
"platform": "XboxTv",
"plugin_map": {
"plugin_name": "homebridge-xbox-tv"
},
"devices": [
{
"name": "XBOXONE",
"host": "10.0.0.62",
"xboxliveid": "FD00B581C41F****",
"volumeControl": false,
"switchInfoMenu": false,
"inputs": [

{
                            "name": "Spotify",
                            "reference": "SpotifyAB.SpotifyMusic-forXbox_zpdnekdrzrea0!App"
                        },
                        {
                            "name": "YouTube",
                            "reference": "GoogleInc.YouTube_yfg5n0ztvskxp!App"
                        },
                        {
                            "name": "Netflix",
                            "reference": "4DF9E0F8.Netflix_mcm4njqhnhss8!App"
                        },
                        {
                            "name": "Telewizja",
                            "reference": "Microsoft.Xbox.LiveTV_8wekyb3d8bbwe!Microsoft.Xbox.LiveTV.Application"
                        },
                        {
                            "name": "Dashboard",
                            "reference": "Xbox.Dashboard_8wekyb3d8bbwe!Xbox.Dashboard.Application"
                        },
                        {
                            "name": "Ustawienia",
                            "reference": "Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application"
                        },
                        {
                            "name": "Sklep",
                            "reference": "Microsoft.WindowsStore_8wekyb3d8bbwe!App"
                        },
                        {
                            "name": "Akcesoria",
                            "reference": "Microsoft.XboxDevices_8wekyb3d8bbwe!App"
                        },
                        {
                            "name": "Microsoft Edge",
                            "reference": "Microsoft.MicrosoftEdge_8wekyb3d8bbwe!MicrosoftEdge"
                        },
                        {
                            "name": "Airserver",
                            "reference": "F3F176BD.53203526D8F6C_p8qzvses5c8me!AirServer"
                        },
                        {
                            "name": "Gears of War 5",
                            "reference": "Microsoft.HalifaxBaseGame_8wekyb3d8bbwe!HalifaxGameShip"
                        },
                        {
                            "name": "Fortnite",
                            "reference": "Fortnite_d5xxtpggmzx6p!AppFortnite"
                        },
                        {
                            "name": "Minecraft",
                            "reference": "Microsoft.MinecraftUWPConsole_8wekyb3d8bbwe!App"
                        },
                        {
                            "name": "Plex",
                            "reference": "CAF9E577.PlexforXbox_aam28m9va5cke!App"
                        },
                        {
                            "name": "Bluray",
                            "reference": "Microsoft.BlurayPlayer_8wekyb3d8bbwe!Xbox.BlurayPlayer.Application"
                        },
                        {
                            "name": "COD WII",
                            "reference": "shg2SubmissionENFR_ht1qfjb0gaftw!S2Boot"
                        },
                        {
                            "name": "COD WZ",
                            "reference": "iw8Submission-EN-FR_ht1qfjb0gaftw!iw8"
                        },
                        {
                            "name": "GTA V",
                            "reference": "GTA-V_vesz1v3mcwykm!GTAV"
                        },
                        {
                            "name": "All4",
                            "reference": "CHANNELFOURTELEVISIONCOMP.All4_e1252dwpj85a4!vstest.executionengine.universal.App"
                        },
                        {
                            "name": "Amazon Prime",
                            "reference": "AmazonVideo.AmazonVideoUK_pwbj9vvecjh7j!App"
                        },
                        {
                            "name": "Disney",
                            "reference": "Disney.37853FC22B2CE_6rarf9sa4v8jt!App"
                        },
                        {
                            "name": "BBC iPlayer",
                            "reference": "BBCMobileApps.BBCIPLAYER_wzgfedwv7gft2!App"
                        }

]
}
]
}
```

**Comment by kvanwagoner** [comment]
That's what I used for my config ... I copied it and then just removed the apps/games I don't have and updated the IP and Xbox live info.

**Comment by Haglerd** [comment]
Any luck getting this to work, should I give it a shot or is it not ready for prime time yet?

**Comment by RoyHobbs99** [comment]
Gave up after multiple attempts.

**Comment by Haglerd** [comment]
I'm going to try version 0.3.12 as someone else said that works. I like the concept and it would be nice if I could figure this out lol.

**Comment by RoyHobbs99** [comment]
I tried the smart glass Xbox plugin and it worked the first time.  Maybe I’ll try the other version of this one tomorrow

**Comment by Haglerd** [comment]
I got the same issues there. The state would never work or update. Sometimes it would, but only once. Then you could only turn it off and not on... I wish I knew more about the code so I could just update it myself lol

**Comment by grzegorz914** [comment]
This plugin working like a harm I see no any issue in the log. Please post Your configurations, logs, console model, system.

**Comment by jlian** [comment]
I have thew same issue. I can turn off but not on. When I tried turning on I get

```
[7/1/2020, 01:22:33] [XboxTv] Device: 192.168.4.46 Xbox One X, booting failed, error: {
  status: 'error_discovery',
  error: 'Console was not found on network. Probably failed'
}
```

Once it's manually turned on using a controller, this plugin works well. I can use the remote, it can see the app, and it can turn off the Xbox. 

```
[7/1/2020, 01:16:57] [XboxTv] Device: 192.168.4.46 Xbox One X, get current Power state successful, state: ON
[7/1/2020, 01:16:57] [XboxTv] Device: 192.168.4.46 Xbox One X, get current App successful: Settings Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application
[7/1/2020, 01:17:00] [XboxTv] Device: 192.168.4.46 Xbox One X, setRemoteKey successful,  command: down
[7/1/2020, 01:17:14] [XboxTv] Device: 192.168.4.46 Xbox One X, setRemoteKey successful,  command: up
[7/1/2020, 01:17:14] [XboxTv] Device: 192.168.4.46 Xbox One X, setRemoteKey successful,  command: up
[7/1/2020, 01:17:15] [XboxTv] Device: 192.168.4.46 Xbox One X, setRemoteKey successful,  command: up
[7/1/2020, 01:18:07] [XboxTv] Device: 192.168.4.46 Xbox One X, set new Power state successful, new state: OFF
```

Configuration

```
        {
            "devices": [
                {
                    "name": "Xbox One X",
                    "host": "192.168.4.46",
                    "xboxliveid": "FD006DE9XXXXXXXX",
                    "volumeControl": 0,
                    "switchInfoMenu": true,
                    "inputs": [
                        {
                            "name": "Dashboard",
                            "reference": "Xbox.Dashboard_8wekyb3d8bbwe!Xbox.Dashboard.Application",
                            "type": "HOME_SCREEN"
                        },
                        {
                            "name": "Settings",
                            "reference": "Microsoft.Xbox.Settings_8wekyb3d8bbwe!Xbox.Settings.Application",
                            "type": "OTHER"
                        }
                    ]
                }
            ],
            "platform": "XboxTv"
        }
```

Console model: Xbox One X

Power mode: Instant-on

I've double triple checked that the `xboxliveid` matches the "Xbox Live device ID" in console info.

**Comment by grzegorz914** [comment]
Try to PowerON console using Xbox app on iPhone, if not work then U have problem with Your network settings (ruter, firewall, port forwarding)

**Comment by tomvining** [comment]
@grzegorz914 - Firstly, great app, loving it. I was tripped up for a few days as homebridge kept saying my xBox was offline. I found the issue in the xBox settings for another piece of config which needs setting. Would it be possible to amend your readme to include that in Console Settings > Devices & Connections > Remote Features > xBox app Preferences > xBox App should be set to "Allow Connections from any device"? The moment I enabled this the homekit app started to work and all errors disappeared. I didn't get any form of authentication popup prior to setting this. Wondering if this might solve @RoyHobbs99 issue also?
